k8s/Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM docker.io/scylladb/scylla:4.1.6 as base
```

Comment:
at some point also the docker image might go...

Reply:
Any reason not to update this?

Reply:
Yes, scripts in newer version changed, and i'm no longer able to provide where xfs raid device should be mounted. This setup relied on it, and it was possible in scylla 4.1.6.

Reply:
Please add that as a comment into the Dockerfile so people are aware of the breakage and don't go after us to upgrade it. Also mention we are planning to drop this setup.

Reply:
comment added